index.js

```diff
@@ -59,7 +61,30 @@ QueryGenerator.upsertQuery = function(tableName, insertValues, updateValues, whe
 
   console.log(insert + " ON CONFlICT (" + pkCols.join(',') + ") DO UPDATE SET " + onConflictSet + ";");
   return insert + " ON CONFlICT (" + pkCols.join(',') + ") DO UPDATE SET " + onConflictSet + ";";
-}
+};
+
+// The JavaScript number type cannot represent all 64-bit integers--it can only
+// exactly represent integers in the range [-2^53 + 1, 2^53 - 1]. Notably,
+// CockroachDB's unique_rowid() function returns values outside the
+// representable range.
+//
+// We must teach Sequelize's INTEGER and BIGINT types to accept stringified
+// numbers instead of just raw JavaScript numbers; it's otherwise impossible to
+// store a number outside the representable range into a CockroachDB INT column.
+[DataTypes.postgres.INTEGER, DataTypes.postgres.BIGINT].forEach(function (intType) {
+  // Disable escaping so that the returned string is not wrapped in quotes
+  // downstream. Valid integers cannot be dangerous, and we take care to reject
+  // invalid integers.
+  intType.prototype.escape = false;
+
+  intType.prototype.$stringify = function stringify(value) {
```

Comment:
Ahh I see, I guess since that will be a major release we don't need to worry about this breaking?